### PR TITLE
v2ray: update to 5.4.1

### PR DESCRIPTION
**Pull request by ipkalm**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-musl
  - i686
  - i686-musl
  - aarch64
  - aarch64-musl
  - armv7l
  - armv7l-musl
  - armv6l
  - armv6l-musl
  - ppc64le

### CODE REVIEW

**Comment by ipkalm** [comment]
Locally I built again with `-Q` arg without any errors for `i686`.

File: srcpkgs/v2ray/INSTALL.msg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Consult /usr/share/doc/v2ray/README.voidlinux for additional configuration
+instructions.
```
Comment (Duncaen) [inline srcpkgs/v2ray/INSTALL.msg:2]: Please don't add noise install messages, this is not required and results in users just ignoring them all together.

File: srcpkgs/v2ray/REMOVE [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#
+# This script removes /etc/v2ray directory only if it is empty
+# and not during package update
+#
+
+if [ "$UPDATE" = "no" ]; then
+        case "$ACTION" in
+        post)
+                [ -d etc/v2ray ] && rmdir etc/v2ray 2>/dev/null
```
Comment (Duncaen) [inline srcpkgs/v2ray/REMOVE:9]: This should not be required. xbps does that.

File: srcpkgs/v2ray/files/v2ray/run [hunk 3]
```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
 exec 2>&1
 
-exec chpst -u _v2ray v2ray run -config=/etc/v2ray/config.json 2>&1
+[ -r conf ] && . ./conf
+
+exec chpst -e /etc/v2ray/env -u _v2ray v2ray run ${OPTS:- -config=/etc/v2ray/config.json} 2>&1
```
Comment (Duncaen) [inline srcpkgs/v2ray/files/v2ray/run:6]: the conf file can already export environment variables. for consistency with other service please remove it.

File: srcpkgs/v2ray/template [hunk 4]
```diff
@@ -49,4 +50,7 @@ post_install() {
 	vmkdir etc/v2ray
 	vcopy release/config/*.json etc/v2ray/
 	vsv v2ray
+
+	# Void-specific documentation.
+	vdoc "${FILESDIR}/README.voidlinux"
```
Comment (Duncaen) [inline srcpkgs/v2ray/template:55]: I feel like the readme is pointless if the service follows how every other service is configured.

**Review by Duncaen — CHANGES_REQUESTED** [review]
See inline comments.

**Pull request closed without merging**